src/pudl/extract/xbrl.py

```diff
@@ -128,10 +134,8 @@ def convert_form(
 
         run_main(
             instance_path=filings_archive,
-            sql_path=PudlPaths()
-            .sqlite_db(f"ferc{form.value}_xbrl")
-            .removeprefix("sqlite:///"),  # Temp hacky solution
-            clobber=clobber,
+            sql_path=sql_path,
+            clobber=False,  # if we set clobber=True, clobbers on *every* call to run_main
```

Comment:
It seems like this makes the `clobber` option non-functional, at least in this location. The idea with `clobber` is to functionally `rm -f the_database.sqlite` before starting a new ETL run. Maybe it should only exist at the highest level in the `ferc_to_sqlite` script and actually unlink the paths if set -- and then everything in here can just be like "You gave me a DB URL and I'm going to write to it. Not my problem if there was already something incompatible there."

Reply:
I think the `unlink()` on line 92 does the `rm -f` you're looking for?

Reply:
Previously when I failed to clobber, the XBRL extractor would write duplicate data -  when I ran `xbrl_to_sqlite` multiple times I still only had one copy of the data afterwards. I guess it could have not written any data at all, the later times. I can run again and see if the database is gone between the `unlink()` and the database write.

Reply:
Yeah, the database gets deleted at the `unlink()` step if `clobber` is set to `True`.

Reply:
DELETED. Strongbad would be proud. I guess the other half of the logic we've previously associated with `--clobber` is that if the DB already exists and you attempt to run the ETL, it bails out early, to avoid unintentionally modifying an existing DB. It's not trying to be anything fancy -- just a binary switch "Do you want me to make something new and blow everything that exists away if it's there? Or do you expect that there's a clean slate, and there should be no possible conflicts.

Reply:
I don't think this is the current behavior though. If you don't say clobber and there is an existing DB what happens?

Reply:
In the past if you don't say clobber and there's an existing DB, we actually just append to the existing DB. Which is pretty bad. I'll add a quick check to just bail out if there's an existing DB instead.